metadata-io/src/main/java/com/linkedin/metadata/params/ExtraIngestParamsHelper.java

```diff
@@ -0,0 +1,33 @@
+package com.linkedin.metadata.params;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+public class ExtraIngestParamsHelper {
+
+    // condUpdate String format: "urn1+AspectName1=createdOn1;urn2+AspectName2=createdOn2;...;urnN+AspectNameN=createdOnN"
+
+    public static Map<String, Long> extractCondUpdate(String condUpdate) {
```

Comment:
This probably needs a JavaDoc to explain the format of the condUpdate parameter. We've explained the `urn1+AspectName1` part but not the format of `createdOn1`, for example.

I'd advise to make an explicit reference to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Unmodified-Since and explain that we are NOT following this standard (because it's very hard to follow) but we could follow it in future if some HTTP clients stick religiously to the standard.